.vscode/launch.json

```diff
@@ -5,6 +5,22 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Run neovim extension",
+      "request": "attach",
+      "processName": "node.exe",
+      "skipFiles": ["<node_internals>/**"],
+      "preLaunchTask": "Build neovim extension only",
+      "type": "node"
+    },
+    {
+      "name": "Run neovim extension tests",
+      "request": "attach",
+      "processName": "node.exe",
+      "skipFiles": ["<node_internals>/**"],
+      "preLaunchTask": "Build neovim extension tests only",
+      "type": "node"
+    },
```

Comment:
would be good to have docs on how to do local development in contributing docs. Esp worth mentioning there that you need to manually kill neovim when you're done running, right?

Reply:
done